packages/components/src/sortTabs/SortTab.js

```diff
@@ -59,13 +60,17 @@ class SortTab extends React.Component<Props> {
         <View
           style={[
             styles.container,
-            isFirst && styles.containerFirst,
-            isLast && styles.containerLast,
-            isActive && styles.containerActive,
+            mobileLayout ? styles.heightMobile : styles.heightWeb,
```

Comment:
I'm slightly worried how big this component become and if those conditions will be readable after a while.
What about splitting it into smaller pieces? For example this one can be something like `SortTabWrapper` which will render `View` with different styles based on `mobileLayout` prop. So it can be slightly more readable. Please just consider it, if it will actually make component more complex (lot of props passing etc.) I can probably live even with those complex conditions.

Reply:
I think that for now, it's okay, there are not many conditions and they won't grow that much...